### PR TITLE
Add form assistant

### PR DESCRIPTION
**Pull request by LittleLittleCloud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,11 +37,16 @@ Welcome to the official home for .NET samples demonstrating how to use AI in you
 | REF | RAG                                         |  [Coming Soon](.)
 | REF | Prompts / Prompt Engineering                |  [Coming Soon](.)
 
+### LLM Agents
```
Comment (JakeRadMSFT) [inline README.md:40]:
```suggestion
### Agent Examples

~~ Text to describe Agents ~~
```

File: README.md [hunk 2]
```diff
@@ -37,11 +37,16 @@ Welcome to the official home for .NET samples demonstrating how to use AI in you
...
+
+|  #  | Topic                                       | GitHub Link                               |
+|-----|---------------------------------------------|-------------------------------------------|
+| REF | Form Assistant                              |  [Form Assistant](./src/llm-agents/form-assistant/README.md)
```
Comment (JakeRadMSFT) [inline README.md:44]:
```suggestion
| REF | Form Assistant                              |  [Form Assistant](./src/agents/auto-gen/form-assistant/README.md)
```

Comment (LittleLittleCloud) [reply README.md:46]: Maybe llm-agents is more proper here as autogen is not the only framework that provides agent solution?

**Comment by giuliohome** [comment]
Very interesting: I was looking for an example of AI form filling Chat Assistant like that!
Why wasn't this merged?!?

**Comment by giuliohome** [comment]
I have also introduced a small variation [here in my fork](https://github.com/giuliohome-org/ai-samples/blob/u/xiaoyun/addAutoGenExample/src/llm-agents/autogen/form-assistant), where I replaced the automatic user agent with a human user typing their responses directly in the terminal.

**Pull request closed without merging**